packages/xtee-misp2-postgresql/install-misp2-postgresql.sh

```diff
@@ -290,12 +290,12 @@ then
 	if ! (echo "$existing_users" | tr " " "\n" | grep -wq "$username")
 	then
 		echo "Adding new user '$username'"
-		$pgsql_dir/createuser  -p $pgport -U postgres -P -A -D $username
+		$pgsql_dir/createuser  -p $pgport -U postgres -P -D $username
 		if [ ! "$PIPESTATUS" = "0" ];
 		then
 			echo "Cannot add new user '$username'"
 			echo "If user does not exist in the database then create him by running: "
```

Comment:
I would change this a little bit to make it gender neutral:

> If user does not exist in the database then create them by running:

It's called "singular they":

https://en.wikipedia.org/wiki/Singular_they

Reply:
In future releases, we should review the whole documentation for similar cases.

Reply:
Updated.